src/amo/routes.js

```diff
@@ -14,6 +15,7 @@ export default (
     <IndexRoute component={Home} />
     <Route path="addon/:slug/" component={DetailPage} />
     <Route path="addon/:slug/review/:reviewId/" component={AddonReview} />
+    <Route path="categories/:addonType/" component={CategoriesPage} />
```

Comment:
Oh, actually it's not, just realised.

`categories/:addonType/` lists all the categories for a given addonType (extension, theme, etc.)

I'll add the `s` though 👍